crest/Assets/Crest/Crest/Scripts/Editor/BuildProcessor.cs

```diff
@@ -60,6 +60,11 @@ public void OnProcessShader(Shader shader, ShaderSnippetData snippet, IList<Shad
             {
                 ProcessUnderwaterShader(shader, data);
             }
+
+            if (shader.name == "Crest/Underwater Curtain")
```

Comment:
i think downstream this code calls a function to check if its one of the underwater shaders. doesnt really matter but just a note in case you want to pull it up stream to here.

Reply:
Done. Thanks.